[wampcc] Build failure

Package: wampcc[core,examples]:x64-linux -> 2019-09-04#4

**Host Environment**

- Host: x64-linux
- Compiler: GNU 11.3.0
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: eea75e5b0 2023-07-03 (2 hours ago)

**To Reproduce**

`vcpkg x-set-installed wampcc[core,examples]`

**Failure logs**

```
-- Downloading https://github.com/darrenjs/wampcc/archive/43d10a7ccf37ec1b895742712dd4a05577b73ff1.tar.gz -> darrenjs-wampcc-43d10a7ccf37ec1b895742712dd4a05577b73ff1.tar.gz...
-- Extracting source /mnt/vcpkg-ci/downloads/darrenjs-wampcc-43d10a7ccf37ec1b895742712dd4a05577b73ff1.tar.gz
-- Using source at /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean
-- Configuring x64-linux
-- Building x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /mnt/vcpkg-ci/downloads/tools/cmake-3.25.1-linux/cmake-3.25.1-linux-x86_64/bin/cmake --build . --config Debug --target install -- -v -j33
    Working Directory: /mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg
    See logs for more information:
      /mnt/vcpkg-ci/buildtrees/wampcc/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  /mnt/vcpkg-ci/installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  /mnt/vcpkg-ci/installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/wampcc/portfile.cmake:28 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/mnt/vcpkg-ci/buildtrees/wampcc/install-x64-linux-dbg-out.log</summary>

```
[1/78] /usr/bin/cc  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -g -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/__/__/3rdparty/apache/base64.c.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/__/__/3rdparty/apache/base64.c.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/__/__/3rdparty/apache/base64.c.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/apache/base64.c
[2/78] /usr/bin/cc  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -g -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/__/__/3rdparty/http_parser/http_parser.c.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/__/__/3rdparty/http_parser/http_parser.c.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/__/__/3rdparty/http_parser/http_parser.c.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/http_parser/http_parser.c
[3/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/error.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/error.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/error.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/error.cc
[4/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/socket_address.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/socket_address.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/socket_address.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/socket_address.cc
[5/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/platform.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/platform.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/platform.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/platform.cc
[6/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/json/CMakeFiles/wampcc_json_static.dir/vendors.cc.o -MF libs/json/CMakeFiles/wampcc_json_static.dir/vendors.cc.o.d -o libs/json/CMakeFiles/wampcc_json_static.dir/vendors.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/json/vendors.cc
[7/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/json/CMakeFiles/wampcc_json_static.dir/json_pointer.cc.o -MF libs/json/CMakeFiles/wampcc_json_static.dir/json_pointer.cc.o.d -o libs/json/CMakeFiles/wampcc_json_static.dir/json_pointer.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/json/json_pointer.cc
[8/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/ssl.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/ssl.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/ssl.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/ssl.cc
/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/ssl.cc: In constructor ‘wampcc::ssl_context::ssl_context(wampcc::logger&, const wampcc::ssl_config&)’:
/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/ssl.cc:41:23: warning: ‘int ERR_load_BIO_strings()’ is deprecated: Since OpenSSL 3.0 [-Wdeprecated-declarations]
   41 |   ERR_load_BIO_strings();
      |   ~~~~~~~~~~~~~~~~~~~~^~
In file included from /mnt/vcpkg-ci/installed/x64-linux/include/openssl/cryptoerr.h:17,
                 from /mnt/vcpkg-ci/installed/x64-linux/include/openssl/crypto.h:38,
                 from /mnt/vcpkg-ci/installed/x64-linux/include/openssl/bio.h:30,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include/wampcc/ssl.h:14,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/ssl.cc:8:
/mnt/vcpkg-ci/installed/x64-linux/include/openssl/cryptoerr_legacy.h:31:27: note: declared here
   31 | OSSL_DEPRECATEDIN_3_0 int ERR_load_BIO_strings(void);
      |                           ^~~~~~~~~~~~~~~~~~~~
[9/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/utils.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/utils.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/utils.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/utils.cc
[10/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/http_parser.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/http_parser.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/http_parser.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/http_parser.cc
[11/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/rawsocket_protocol.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/rawsocket_protocol.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/rawsocket_protocol.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/rawsocket_protocol.cc
[12/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/websocketpp_impl.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/websocketpp_impl.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/websocketpp_impl.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/websocketpp_impl.cc
[13/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/io_loop.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/io_loop.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/io_loop.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/io_loop.cc
[14/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/kernel.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/kernel.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/kernel.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/kernel.cc
[15/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/json/CMakeFiles/wampcc_json_static.dir/json.cc.o -MF libs/json/CMakeFiles/wampcc_json_static.dir/json.cc.o.d -o libs/json/CMakeFiles/wampcc_json_static.dir/json.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/json/json.cc
In file included from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/object.hpp:15,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack.hpp:10,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/json/msgpack_serialiser.h:13,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/json/json.cc:2:
/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/v1/object.hpp: In function ‘void msgpack::v1::operator<<(msgpack::v2::object&, const msgpack_object&)’:
/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/v1/object.hpp:647:16: warning: ‘void* memcpy(void*, const void*, size_t)’ copying an object of non-trivial type ‘struct msgpack::v2::object’ from an array of ‘const msgpack_object’ {aka ‘const struct msgpack_object’} [-Wclass-memaccess]
  647 |     std::memcpy(&o, &v, sizeof(v));
      |     ~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~
In file included from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/object_fwd.hpp:17,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/v1/adaptor/adaptor_base_decl.hpp:14,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/adaptor/adaptor_base_decl.hpp:13,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/adaptor/adaptor_base.hpp:13,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/v1/object_decl.hpp:16,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/object_decl.hpp:14,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/object.hpp:13,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack.hpp:10,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/json/msgpack_serialiser.h:13,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/json/json.cc:2:
/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/v2/object_fwd.hpp:23:8: note: ‘struct msgpack::v2::object’ declared here
   23 | struct object : v1::object {
      |        ^~~~~~
[16/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/protocol.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/protocol.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/protocol.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/protocol.cc
[17/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/ssl_socket.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/ssl_socket.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/ssl_socket.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/ssl_socket.cc
[18/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/json/CMakeFiles/wampcc_json_static.dir/msgpack_serialiser.cc.o -MF libs/json/CMakeFiles/wampcc_json_static.dir/msgpack_serialiser.cc.o.d -o libs/json/CMakeFiles/wampcc_json_static.dir/msgpack_serialiser.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/json/msgpack_serialiser.cc
In file included from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/object.hpp:15,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack.hpp:10,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/json/msgpack_serialiser.h:13,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/json/msgpack_serialiser.cc:8:
/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/v1/object.hpp: In function ‘void msgpack::v1::operator<<(msgpack::v2::object&, const msgpack_object&)’:
/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/v1/object.hpp:647:16: warning: ‘void* memcpy(void*, const void*, size_t)’ copying an object of non-trivial type ‘struct msgpack::v2::object’ from an array of ‘const msgpack_object’ {aka ‘const struct msgpack_object’} [-Wclass-memaccess]
  647 |     std::memcpy(&o, &v, sizeof(v));
      |     ~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~
In file included from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/object_fwd.hpp:17,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/v1/adaptor/adaptor_base_decl.hpp:14,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/adaptor/adaptor_base_decl.hpp:13,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/adaptor/adaptor_base.hpp:13,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/v1/object_decl.hpp:16,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/object_decl.hpp:14,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/object.hpp:13,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack.hpp:10,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/json/msgpack_serialiser.h:13,
                 from /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/json/msgpack_serialiser.cc:8:
/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/msgpack-c/include/msgpack/v2/object_fwd.hpp:23:8: note: ‘struct msgpack::v2::object’ declared here
   23 | struct object : v1::object {
      |        ^~~~~~
[19/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/demo_embedded_router_ssl.dir/basic/demo_embedded_router_ssl.cc.o -MF examples/CMakeFiles/demo_embedded_router_ssl.dir/basic/demo_embedded_router_ssl.cc.o.d -o examples/CMakeFiles/demo_embedded_router_ssl.dir/basic/demo_embedded_router_ssl.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/basic/demo_embedded_router_ssl.cc
[20/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/event_loop.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/event_loop.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/event_loop.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/event_loop.cc
[21/78] : && /mnt/vcpkg-ci/downloads/tools/cmake-3.25.1-linux/cmake-3.25.1-linux-x86_64/bin/cmake -E rm -f libs/json/libwampcc_json.a && /usr/bin/ar qc libs/json/libwampcc_json.a  libs/json/CMakeFiles/wampcc_json_static.dir/json.cc.o libs/json/CMakeFiles/wampcc_json_static.dir/json_pointer.cc.o libs/json/CMakeFiles/wampcc_json_static.dir/msgpack_serialiser.cc.o libs/json/CMakeFiles/wampcc_json_static.dir/vendors.cc.o && /usr/bin/ranlib libs/json/libwampcc_json.a && :
[22/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/demo_embedded_router.dir/basic/demo_embedded_router.cc.o -MF examples/CMakeFiles/demo_embedded_router.dir/basic/demo_embedded_router.cc.o.d -o examples/CMakeFiles/demo_embedded_router.dir/basic/demo_embedded_router.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/basic/demo_embedded_router.cc
[23/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/basic_embedded_router.dir/basic/basic_embedded_router.cc.o -MF examples/CMakeFiles/basic_embedded_router.dir/basic/basic_embedded_router.cc.o.d -o examples/CMakeFiles/basic_embedded_router.dir/basic/basic_embedded_router.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/basic/basic_embedded_router.cc
[24/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/basic_subscriber.dir/basic/basic_subscriber.cc.o -MF examples/CMakeFiles/basic_subscriber.dir/basic/basic_subscriber.cc.o.d -o examples/CMakeFiles/basic_subscriber.dir/basic/basic_subscriber.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/basic/basic_subscriber.cc
[25/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/basic_caller.dir/basic/basic_caller.cc.o -MF examples/CMakeFiles/basic_caller.dir/basic/basic_caller.cc.o.d -o examples/CMakeFiles/basic_caller.dir/basic/basic_caller.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/basic/basic_caller.cc
[26/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/basic_callee.dir/basic/basic_callee.cc.o -MF examples/CMakeFiles/basic_callee.dir/basic/basic_callee.cc.o.d -o examples/CMakeFiles/basic_callee.dir/basic/basic_callee.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/basic/basic_callee.cc
[27/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/basic_callee_ssl.dir/basic/basic_callee_ssl.cc.o -MF examples/CMakeFiles/basic_callee_ssl.dir/basic/basic_callee_ssl.cc.o.d -o examples/CMakeFiles/basic_callee_ssl.dir/basic/basic_callee_ssl.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/basic/basic_callee_ssl.cc
[28/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/basic_publisher.dir/basic/basic_publisher.cc.o -MF examples/CMakeFiles/basic_publisher.dir/basic/basic_publisher.cc.o.d -o examples/CMakeFiles/basic_publisher.dir/basic/basic_publisher.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/basic/basic_publisher.cc
[29/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/rpc_man.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/rpc_man.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/rpc_man.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/rpc_man.cc
[30/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/pubsub_man.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/pubsub_man.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/pubsub_man.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/pubsub_man.cc
[31/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/basic_authentication_provider.dir/authentication/basic_authentication_provider.cc.o -MF examples/CMakeFiles/basic_authentication_provider.dir/authentication/basic_authentication_provider.cc.o.d -o examples/CMakeFiles/basic_authentication_provider.dir/authentication/basic_authentication_provider.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/authentication/basic_authentication_provider.cc
[32/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/ticket_authentication_provider.dir/authentication/ticket_authentication_provider.cc.o -MF examples/CMakeFiles/ticket_authentication_provider.dir/authentication/ticket_authentication_provider.cc.o.d -o examples/CMakeFiles/ticket_authentication_provider.dir/authentication/ticket_authentication_provider.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/authentication/ticket_authentication_provider.cc
[33/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/tcp_socket.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/tcp_socket.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/tcp_socket.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/tcp_socket.cc
[34/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/data_model.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/data_model.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/data_model.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/data_model.cc
[35/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/message_subscriber.dir/message_server/message_subscriber.cc.o -MF examples/CMakeFiles/message_subscriber.dir/message_server/message_subscriber.cc.o.d -o examples/CMakeFiles/message_subscriber.dir/message_server/message_subscriber.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/message_server/message_subscriber.cc
FAILED: examples/CMakeFiles/message_subscriber.dir/message_server/message_subscriber.cc.o 
/usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/message_subscriber.dir/message_server/message_subscriber.cc.o -MF examples/CMakeFiles/message_subscriber.dir/message_server/message_subscriber.cc.o.d -o examples/CMakeFiles/message_subscriber.dir/message_server/message_subscriber.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/message_server/message_subscriber.cc
/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/message_server/message_subscriber.cc: In function ‘std::unique_ptr<wampcc::tcp_socket> get_tcp_connection(const char*, int, wampcc::kernel&, std::chrono::milliseconds)’:
/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/message_server/message_subscriber.cc:79:25: error: ‘sleep_for’ is not a member of ‘std::this_thread’
   79 |       std::this_thread::sleep_for(interval);
      |                         ^~~~~~~~~
/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/message_server/message_subscriber.cc: In function ‘int main_impl(int, char**)’:
/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/message_server/message_subscriber.cc:161:23: error: ‘sleep_for’ is not a member of ‘std::this_thread’
  161 |     std::this_thread::sleep_for(std::chrono::seconds(1));
      |                       ^~~~~~~~~
[36/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/disclose_authorization_provider.dir/authorization/disclose_authorization_provider.cc.o -MF examples/CMakeFiles/disclose_authorization_provider.dir/authorization/disclose_authorization_provider.cc.o.d -o examples/CMakeFiles/disclose_authorization_provider.dir/authorization/disclose_authorization_provider.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/authorization/disclose_authorization_provider.cc
[37/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/wamp_router.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/wamp_router.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/wamp_router.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/wamp_router.cc
[38/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/external_authentication_provider.dir/authentication/external_authentication_provider.cc.o -MF examples/CMakeFiles/external_authentication_provider.dir/authentication/external_authentication_provider.cc.o.d -o examples/CMakeFiles/external_authentication_provider.dir/authentication/external_authentication_provider.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/authentication/external_authentication_provider.cc
[39/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/basic_authorization_provider.dir/authorization/basic_authorization_provider.cc.o -MF examples/CMakeFiles/basic_authorization_provider.dir/authorization/basic_authorization_provider.cc.o.d -o examples/CMakeFiles/basic_authorization_provider.dir/authorization/basic_authorization_provider.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/authorization/basic_authorization_provider.cc
[40/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/websocket_protocol.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/websocket_protocol.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/websocket_protocol.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/websocket_protocol.cc
[41/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/basic_server.dir/basic/basic_server.cc.o -MF examples/CMakeFiles/basic_server.dir/basic/basic_server.cc.o.d -o examples/CMakeFiles/basic_server.dir/basic/basic_server.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/basic/basic_server.cc
[42/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/ticket_caller.dir/authentication/ticket_caller.cc.o -MF examples/CMakeFiles/ticket_caller.dir/authentication/ticket_caller.cc.o.d -o examples/CMakeFiles/ticket_caller.dir/authentication/ticket_caller.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/authentication/ticket_caller.cc
[43/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/helper.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/helper.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/helper.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/helper.cc
[44/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/disclose_caller.dir/authorization/disclose_caller.cc.o -MF examples/CMakeFiles/disclose_caller.dir/authorization/disclose_caller.cc.o.d -o examples/CMakeFiles/disclose_caller.dir/authorization/disclose_caller.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/authorization/disclose_caller.cc
[45/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/disclose_callee.dir/authorization/disclose_callee.cc.o -MF examples/CMakeFiles/disclose_callee.dir/authorization/disclose_callee.cc.o.d -o examples/CMakeFiles/disclose_callee.dir/authorization/disclose_callee.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/authorization/disclose_callee.cc
[46/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/disclose_publisher.dir/authorization/disclose_publisher.cc.o -MF examples/CMakeFiles/disclose_publisher.dir/authorization/disclose_publisher.cc.o.d -o examples/CMakeFiles/disclose_publisher.dir/authorization/disclose_publisher.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/authorization/disclose_publisher.cc
[47/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/disclose_subscriber.dir/authorization/disclose_subscriber.cc.o -MF examples/CMakeFiles/disclose_subscriber.dir/authorization/disclose_subscriber.cc.o.d -o examples/CMakeFiles/disclose_subscriber.dir/authorization/disclose_subscriber.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/authorization/disclose_subscriber.cc
[48/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/ssl_client.dir/ssl_socket/ssl_client.cc.o -MF examples/CMakeFiles/ssl_client.dir/ssl_socket/ssl_client.cc.o.d -o examples/CMakeFiles/ssl_client.dir/ssl_socket/ssl_client.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/ssl_socket/ssl_client.cc
[49/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/ssl_server.dir/ssl_socket/ssl_server.cc.o -MF examples/CMakeFiles/ssl_server.dir/ssl_socket/ssl_server.cc.o.d -o examples/CMakeFiles/ssl_server.dir/ssl_socket/ssl_server.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/ssl_socket/ssl_server.cc
[50/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/message_server.dir/message_server/message_server.cc.o -MF examples/CMakeFiles/message_server.dir/message_server/message_server.cc.o.d -o examples/CMakeFiles/message_server.dir/message_server/message_server.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/message_server/message_server.cc
[51/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT examples/CMakeFiles/router.dir/wamp_router/router.cc.o -MF examples/CMakeFiles/router.dir/wamp_router/router.cc.o.d -o examples/CMakeFiles/router.dir/wamp_router/router.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/examples/wamp_router/router.cc
[52/78] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/wampcc/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/include -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty/websocketpp -I/mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/3rdparty -isystem /mnt/vcpkg-ci/installed/x64-linux/include -fPIC -Wall -g -std=gnu++11 -MD -MT libs/wampcc/CMakeFiles/wampcc_static.dir/wamp_session.cc.o -MF libs/wampcc/CMakeFiles/wampcc_static.dir/wamp_session.cc.o.d -o libs/wampcc/CMakeFiles/wampcc_static.dir/wamp_session.cc.o -c /mnt/vcpkg-ci/buildtrees/wampcc/src/5577b73ff1-2b13c67405.clean/libs/wampcc/wamp_session.cc
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

--- a/ports/wampcc/gcc-11.patch
+++ b/ports/wampcc/gcc-11.patch
@@ -1,0 +1,19 @@
+diff --git a/examples/message_server/message_subscriber.cc b/examples/message_server/message_subscriber.cc
+index ba40a332..4ac75ef7 100644
+--- a/examples/message_server/message_subscriber.cc
++++ b/examples/message_server/message_subscriber.cc
+@@ -7,13 +7,9 @@
+ 
+ #include "wampcc/kernel.h"
+ #include "wampcc/data_model.h"
+-#include "wampcc/tcp_socket.h"
+-#include "wampcc/wamp_session.h"
+ #include "wampcc/rawsocket_protocol.h"
+ 
+-#include <memory>
+-#include <iostream>
+-
++#include <thread>
+ 
+ void on_string_change(const wampcc::string_subscription& sub)
+ {

--- a/ports/wampcc/portfile.cmake
+++ b/ports/wampcc/portfile.cmake
@@ -10,6 +10,8 @@ vcpkg_from_github(
     REF 43d10a7ccf37ec1b895742712dd4a05577b73ff1
     SHA512 e830d26de00e8f5f378145f06691cb16121c40d3bd2cd663fad9a97db37251a11b56053178b619e3a2627f0cd518b6290a8381b26e517a9f16f0246d2f91958e
     HEAD_REF master
+    PATCHES
+        gcc-11.patch # https://github.com/darrenjs/wampcc/commit/d1a8c6dcabcc32e9d9774f306555e9080d871c2f
 )
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS

--- a/ports/wampcc/vcpkg.json
+++ b/ports/wampcc/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "wampcc",
   "version-date": "2019-09-04",
-  "port-version": 4,
+  "port-version": 5,
   "description": "Wampcc is C++ library that implements the Web Application Messaging Protocol (WAMP) protocol.",
   "license": "MIT",
   "supports": "!(windows & arm64)",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -8566,7 +8566,7 @@
     },
     "wampcc": {
       "baseline": "2019-09-04",
-      "port-version": 4
+      "port-version": 5
     },
     "wangle": {
       "baseline": "2023.05.15.00",

--- a/versions/w-/wampcc.json
+++ b/versions/w-/wampcc.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "69c3d159720746a5321085dab7b543c1c95eed85",
+      "version-date": "2019-09-04",
+      "port-version": 5
+    },
+    {
       "git-tree": "acfd463862ba55ebabe1f0b0105da56b965d5b90",
       "version-date": "2019-09-04",
       "port-version": 4

